sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src/Utf8JsonWriterExtensions.cs

```diff
@@ -0,0 +1,90 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Text.Json;
+using Azure.Core;
+
+namespace Azure.AI.MetricsAdvisor
+{
+    internal static class Utf8JsonWriterExtensions
```

Comment:
Convenience methods to make serialization code shorter. We can reduce:
```cs
if (value != null)
{
    writer.WritePropertyName(...);
    writer.Write<Something>Value(...);
}
else
{
  writer.WriteNull(propertyName);
}
```
to a single line.